### PR TITLE
Log all test results separately to BigQuery

### PR DESCRIPTION
**Pull request by jessebrennan**

### CODE REVIEW

File: test/bq.py [hunk 1]
```diff
@@ -30,11 +28,45 @@ def list_table(self, table_id, limit=10):
...
+        self.create_table(table_id, schema)
+
+    def log_test_results(self, test_name, status, timestamp, create=False):
+        table_id = f'platform-dev-178517.bdc.integration_tests_{test_name}'
+        if status == 'success':
+            field = 'u'
+        elif status in ('failure', 'error'):
+            field = 'd'
+        elif status == 'skip':
+            field = 'm'
```
Comment (DailyDreaming) [inline test/bq.py:55]: Hmmm... the `m` for maintenance should never be logged currently.  This should be removed and a comment left instead.

Skipped tests should simply not send a row to BigQuery at all, ideally.

File: test/bq.py [hunk 2]
```diff
@@ -30,11 +28,45 @@ def list_table(self, table_id, limit=10):
...
+        else:
+            raise ValueError(f'Unexpected status: {status!r} for test: {test_name!r}')
```
Comment (DailyDreaming) [inline test/bq.py:57]: I hadn't seen `!r` before.  That's kind of nice.

**Review by DailyDreaming — COMMENTED** [review]
Looks good overall!  Just one comment about `m`.

File: test/bq.py (hunk 2 above)
Comment (jessebrennan) [reply test/bq.py]: It's nice for strings because it adds quotes around them which makes it easy to distinguish the interpolated value.

**Review by DailyDreaming — APPROVED** [review]
LGTM!

**Pull request merged**